test/client/spec/components/grid.spec.jsx

```diff
@@ -150,193 +311,331 @@ describe("components/grid", () => {
-      gutter: "16px"
-    });
+    const grid = (
+      <Grid gutter="24px">
+        <Cell>
+          <p>testing</p>
+        </Cell>
+        <Cell>
+          <p>testing!</p>
+        </Cell>
+        <Cell>
+          <p>testing?</p>
+        </Cell>
+      </Grid>
+    );
+    const cells = resolveCells(grid.props);
 
-    for (const style of extractBreakpointStyles(result)) {
+    testCellsAllBreakpoints(cells, (style) => {
       expect(style.flexBasis).to.have.string("calc(33.333");
       expect(style.flexBasis).to.have.string("- 16px");
-    }
+    });
   });
 
+
   it("should allow customizable fluid gutters", () => {
-    const result = resolveCellStyles({
-      ...Grid.defaultProps,
-      gutter: "2%"
-    });
+    const grid = (
+      <Grid gutter="2%">
+        <Cell>
+          <p>testing</p>
+        </Cell>
+        <Cell>
+          <p>testing!</p>
+        </Cell>
+        <Cell>
+          <p>testing?</p>
+        </Cell>
+      </Grid>
+    );
+    const cells = resolveCells(grid.props);
 
-    for (const style of extractBreakpointStyles(result)) {
+    testCellsAllBreakpoints(cells, (style) => {
       expect(style.flexBasis).to.have.string("calc(33.333");
-      expect(style.flexBasis).to.have.string("- 2%");
-    }
-  });
 
-  it("should apply custom props style objects", () => {
-    const result = resolveCellStyles({
-      ...Grid.defaultProps,
-      style: {
-        backgroundColor: "blue"
-      }
+      // TODO: this is a magic number. Write tests for gutter calcs
+      expect(style.flexBasis).to.have.string("- 1.3333");
     });
+  });
 
-    expect(result).to.not.be.empty;
-    expect(result.backgroundColor).to.equal("blue");
+
+  it("should apply custom props style objects", () => {
+    const grid = (
+      <Grid>
+        <Cell style={{backgroundColor: "blue"}}>
+          <p>testing</p>
+        </Cell>
+        <Cell>
+          <p>testing!</p>
+        </Cell>
+        <Cell>
+          <p>testing?</p>
+        </Cell>
+      </Grid>
+    );
+    const cells = resolveCells(grid.props);
+    expect(cells[0].props.style.backgroundColor).to.equal("blue");
   });
 
   it("should apply custom props Radium style arrays", () => {
-    const result = resolveCellStyles({
-      ...Grid.defaultProps,
-      style: [{
-        backgroundColor: "blue"
-      }, {
-        padding: "1rem"
-      }]
-    });
-
-    expect(result).to.not.be.empty;
-    expect(result.backgroundColor).to.equal("blue");
-    expect(result.padding).to.equal("1rem");
+    const grid = (
+      <Grid>
+        <Cell style=
+          {[
+            {backgroundColor: "blue"},
+            {padding: "1rem"}
+          ]}
+        >
+          <p>testing</p>
+        </Cell>
+        <Cell>
+          <p>testing!</p>
+        </Cell>
+        <Cell>
+          <p>testing?</p>
+        </Cell>
+      </Grid>
+    );
+    const cells = resolveCells(grid.props);
+    expect(cells[0].props.style.backgroundColor).to.equal("blue");
```

Comment:
GLOBAL: Use `property` or `deep.property` wherever you can -- much better error messages. So here, for example:

```js
expect(cells).to.have.deep.property("[0].props.style.backgroundColor", "blue");
```